src/components/FileTree.tsx

```diff
@@ -478,6 +484,13 @@ function FileTreeContextMenu({
         >
           Delete
         </ContextMenuItem>,
+        <ContextMenuItem
+          data-testid="context-menu-clone"
+          onClick={onClone}
+          hotkey=""
+        >
+          Clone
```

Comment:
Nit: would Duplicate be better than Clone here? Feels like I've seen it more in web apps. Don't think it's super common in cad apps in general

Reply:
Actually onshapre calls it Copy workspace. Which I find even worse?

<img width="668" alt="image" src="https://github.com/user-attachments/assets/ce0c3eba-ff16-479b-8243-e6f6c490fac5" />

Reply:
Aah it's Figma who's got Duplicate in their lingo 
<img width="475" alt="image" src="https://github.com/user-attachments/assets/fc808484-27e9-4032-960a-14add2a495b4" />

Reply:
Agreed!

Reply:
+1 for "Duplicate". It's standard all over macOS.